includes/class-newspack-newsletters-renderer.php

```diff
@@ -145,7 +145,7 @@ function ( $key ) use ( &$attrs ) {
 		);
 
 		if ( isset( $attrs['background-color'] ) ) {
-			$attrs['padding'] = '16px';
+			$attrs['padding'] = '0';
```

Comment:
This makes background-colored paragraphs in columns render incorecctly:

![image](https://user-images.githubusercontent.com/7383192/80377784-0053f780-889c-11ea-8225-61314cc64b9c.png)

Reply:
Should be fixed in https://github.com/Automattic/newspack-newsletters/pull/149/commits/3b3b1cda698db21bcb7106f4beaf4bac8ff8db7f

Reply:
One more thing I forgot about earlier - same issue is with Heading block in a column.

Reply:
Oh yeah. I've been less specific in https://github.com/Automattic/newspack-newsletters/pull/149/commits/d2a5bdd1e3b767c102e000333e595d951159e95f, hopefully it should do the trick.